### DISCUSSION TITLE
The Crossbow, Trident and other some other additions

### DISCUSSION THREAD

**Discussion by Starexify** [Roadhog360/Et-Futurum-Requiem#111] [Ideas]
Just the addition of the Crossbow and/or Trident and it's enchantments
-maybe with the Crossbow can be added the Pillager and Pillager Outpost, without the raids ofc, 
-and with the Trident the drowneds while the copper is already in the game

More Additions:
The Observer
Fletching table and Cartography table (without GUI) same for Loom, just for building
blocks that would be nice to be added: Stonecutter, Grindstone, Smithing table(for netherite)

Plants like the Bamboo with the scaffolding

that's all that is missing from the mod I think. (with the addons and D-Mod)

**Comment by facetorched** [comment]
Observers are arguably the most useful redstone addition ever! Whether it's making a simple sugarcane farm or a dustless 1-wide tileable binary adder, these things are absolutely baller. I second the observer

(of course, fully implemented slimeblock is pretty important for observer's full functionality)

**Comment by huihiuhuai** [comment]
please consider backporting crossbow!
